tutorials/preprocessing/90_eyetracking_data.py

```diff
@@ -101,123 +108,114 @@
-# signal fluctuations due to the pupil response, so a threshold of 1500 is conservative
-# enough to reject epochs only with large artifacts.
+# blink artifacts biasing our analysis, so we have two options: Drop the blink periods
+# from our data during epoching, or interpolate the missing data during the blink
+# periods. For this tutorial, let's interpolate the blink samples:
+
+mne.preprocessing.eyetracking.interpolate_blinks(raw_et, buffer=(0.05, 0.2))
+
+# %%
+# .. note:: By default, :func:`~mne.preprocessing.eyetracking.interpolate_blinks`, will
+#           only interpolate blinks in ``"pupil"`` channels. Setting
+#           ``interpolate_gaze=True`` will also interpolate the blink periods of the
+#           ``"eyegaze"`` channels. Be aware, however, that eye movements can occur
+#           during blinks which makes the gaze data less suitable for interpolation.
+
+# %%
+# Aligning the eye-tracking data with EEG data
+# --------------------------------------------
+#
+# Eye-tracking and EEG data were recorded simultaneously, but on different
+# systems, so we'll need to align the data before we can analyze them together. We can
+# do this using the :func:`~mne.preprocessing.realign_raw` function, which will align
+# the data based on the timing of the shared events that are present in both
+# :class:`~mne.io.Raw` objects. We'll use the shared photodiode events we extracted
+# above, but first we need to convert the event onsets from samples to seconds. Once the
+# data have been aligned, we'll add the EEG channels to the eye-tracking raw object.
+
+# Convert event onsets from samples to seconds
+et_din_times = et_events[:, 0] / raw_et.info["sfreq"]
+eeg_din_times = eeg_events[:, 0] / raw_eeg.info["sfreq"]
+# Align the data
+mne.preprocessing.realign_raw(
+    raw_et, raw_eeg, et_din_times, eeg_din_times, verbose="error"
+)
+# Add EEG channels to the eye-tracking raw object
+raw_et.add_channels([raw_eeg], force_update_info=True)
+
+# Define a few channel groups of interest and plot the data
+frontal = ["E19", "E11", "E4", "E12", "E5"]
+occipital = ["E61", "E62", "E78", "E67", "E72", "E77"]
+pupil = ["pupil_right"]
+picks_idx = mne.pick_channels(
+    raw_et.ch_names, frontal + occipital + pupil, ordered=True
+)
+raw_et.plot(events=et_events, event_id=event_dict, event_color="g", order=picks_idx)
+
+
+# %%
+# Epoching the data
+# -----------------
 
 epochs = mne.Epochs(
-    raw,
-    events,
-    tmin=-0.3,
-    tmax=5,
+    raw_et,
+    events=et_events,
     event_id=event_dict,
+    tmin=-0.3,
+    tmax=3,
     preload=True,
-    reject=dict(pupil=1500),
 )
-epochs.plot(events=events, event_id=event_dict)
+epochs[:8].plot(events=et_events, event_id=event_dict, order=picks_idx)
 
 # %%
 # We can clearly see the prominent decrease in pupil size following the
 # stimulation.
 
 # %%
-# Plot average pupil response
-# ---------------------------
+# Plot the evoked response
+# ------------------------
```

Comment:
I'd consider merging this with the epoching section and giving it a more generic section title like "showing the pupillary light reflex".